Cargo.toml

```diff
@@ -44,6 +44,7 @@ ttrpc = "0.8.0"
 wat = "1.206"
 windows-sys = "0.52"
 serial_test = "2"
+tracing = "0.1"
```

Comment:
Should this be an optional feature in the case downstream shims don't use it? https://github.com/containerd/rust-extensions/issues/260